src/plugins/intel_cpu/src/nodes/executors/fullyconnected_implementations.cpp

```diff
@@ -74,10 +79,20 @@ static const TypeMapping dnnlFCTypeMapping {
     // @todo explicitly cover configuration limitations for oneDNN on ARM
 };
 
+static const TypeMapping aclFCTypeMapping {
+    // {src, wei, bia, dst}                  pt<src, wei, bias, dst>
+    {{_f32 | _f16, _f32 | _f16, _any, _any}, pt(bypass(), bypass(), use<0>(), use<0>())},
```

Comment:
Does ACL really support combinations like act=f32, wei=f16 or vice versa?

Reply:
@dmitry-gorokhov ACL don't support it, but I add convert precision in pre-packing weights